modules/openapi-generator/src/main/resources/kotlin-client/libraries/multiplatform/api.mustache

```diff
@@ -66,19 +70,22 @@ import kotlinx.serialization.internal.StringDescriptor
             {{/hasFormParams}}
             {{/hasBodyParam}}
 
-        val localVariableQuery = mutableMapOf<String, List<String>>()
-        {{#queryParams}}
-        {{{paramName}}}?.apply { localVariableQuery["{{baseName}}"] = {{#isContainer}}toMultiValue(this, "{{collectionFormat}}"){{/isContainer}}{{^isContainer}}listOf("${{{paramName}}}"){{/isContainer}} }
-        {{/queryParams}}
+        {{! TODO: Don't use all the listOf() }}
+        val localVariableQuery = mutableMapOf<String, List<String>?>(
+            {{#queryParams}}
+            "{{baseName}}" to {{#isContainer}}{{{paramName}}}?.let { toMultiValue(it, "{{collectionFormat}}") }{{/isContainer}}{{^isContainer}}listOf("${{{paramName}}}"){{/isContainer}}{{#hasMore}},{{/hasMore}}
+            {{/queryParams}}
+        )
 
-        val localVariableHeaders = mutableMapOf<String, String>()
-        {{#headerParams}}
-        {{{paramName}}}?.apply { localVariableHeaders["{{baseName}}"] = {{#isContainer}}this.joinToString(separator = collectionDelimiter("{{collectionFormat}}")){{/isContainer}}{{^isContainer}}this.toString(){{/isContainer}} }
-        {{/headerParams}}
+        val localVariableHeaders = mutableMapOf<String, String?>(
+            {{#headerParams}}
+            "{{baseName}}" to {{#isContainer}}{{{paramName}}}?.joinToString(separator = collectionDelimiter("{{collectionFormat}}")){{/isContainer}}{{^isContainer}}this?.toString(){{/isContainer}}{{#hasMore}},{{/hasMore}}
+            {{/headerParams}}
+        )
 
         val localVariableConfig = RequestConfig(
             RequestMethod.{{httpMethod}},
-            "{{path}}"{{#pathParams}}.replace("{" + "{{baseName}}" + "}", "${{{paramName}}}"){{/pathParams}},
+            "{{path}}"{{#pathParams}}.replace("{{curly true}}{{baseName}}{{curly}}", "${{{paramName}}}"){{/pathParams}},
```

Comment:
This also seems like a waste, but I now sure, where and if I can transform that URL:
```kotlin
"/jellyfin/Audio/{itemId}/stream".replace("{itemId}", "$itemId").replace("{container}", "$container")
```

Reply:
(it also doesn't generate the curly braces for some reason)

Reply:
I think I can fix this. Let's address this in a separate PR instead so please undo the change.